genai/embeddings/embed_content_text.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def embed_content() -> str:
+    # [START googlegenaisdk_embed_content_text]
```

Comment:
Can we please use `googlegenaisdk_embeddings_docretrieval_with_txt`
```suggestion
    # [START googlegenaisdk_embeddings_docretrieval_with_txt]
```

Reply:
Yes, please. This is a learning from my past.

When we will have 200~300 samples, this is a super helpful thing